app/src/main/java/com/teamfillin/fillin/presentation/map/StudioMapActivity.kt

```diff
@@ -57,54 +70,50 @@ class StudioMapActivity : BindingActivity<ActivityStudioMapBinding>(R.layout.act
-                mapType = NaverMap.MapType.Navi
-                setLayerGroupEnabled(NaverMap.LAYER_GROUP_TRANSIT, false)
-                setLayerGroupEnabled(NaverMap.LAYER_GROUP_BUILDING, true)
-                isNightModeEnabled = true
-                locationSource = trackingLocationSource
-                locationTrackingMode = LocationTrackingMode.Follow
-                addOnLocationChangeListener { location ->
-                    cameraPosition =
-                        CameraPosition(LatLng(location.latitude, location.longitude), 16.0)
-                }
-                uiSettings.run {
-                    isCompassEnabled = false
-                    isScaleBarEnabled = false
-                    isZoomControlEnabled = false
-                    isLocationButtonEnabled = false
-                }
+    override fun onMapReady(naverMap: NaverMap) {
+        activityNaverMap = naverMap.apply {
+            mapType = NaverMap.MapType.Navi
+            setLayerGroupEnabled(NaverMap.LAYER_GROUP_TRANSIT, false)
+            setLayerGroupEnabled(NaverMap.LAYER_GROUP_BUILDING, true)
+            isNightModeEnabled = true
+            locationTrackingMode = LocationTrackingMode.Follow
+            addOnLocationChangeListener { location ->
+                cameraPosition =
+                    CameraPosition(LatLng(location.latitude, location.longitude), 16.0)
+            }
+            minZoom = 6.0
+            maxZoom = 18.0
+            uiSettings.run {
+                isCompassEnabled = false
+                isScaleBarEnabled = false
+                isZoomControlEnabled = false
+                isLocationButtonEnabled = false
             }
-            binding.btnLocation.map = naverMap
-            behavior.state = BottomSheetBehavior.STATE_HIDDEN
-            markerLocationEvent()
         }
 
-        private fun markerLocationEvent() {
-            val marker = Marker()
-            marker.position = LatLng(37.5666805, 126.9784147)
-            marker.icon = OverlayImage.fromResource(R.drawable.ic_place_big)
-            binding.clBottomSheet.visibility = View.VISIBLE
+        binding.clBottomSheet.visibility = View.VISIBLE
+        markerLocationEvent()
+        binding.btnLocation.map = naverMap
+    }
 
-            marker.setOnClickListener {
-                behavior.state = BottomSheetBehavior.STATE_COLLAPSED
-                true
-            }
-            marker.map = activityNaverMap
+    private fun markerLocationEvent() {
+        val markers = mutableListOf<Marker>()
+
+        lifecycleScope.launch {
+            service.getWholeStudio().await()
+                .data.studio.forEach {
+                    markers += Marker().apply {
+                        position = LatLng(it.lati, it.long)
+                        icon = OverlayImage.fromResource(R.drawable.ic_place_big)
+                        this.map = activityNaverMap
+                    }
+                }
```

Comment:
```suggestion
    runCatching {
        service.getWholeStudio().await()
    }.onSuccess {
        it.data.studio.forEach {
                Marker().run {
                    position = LatLng(it.lati, it.long)
                    icon = OverlayImage.fromResource(R.drawable.ic_place_big)
                    this.map = activityNaverMap
            }
        }
    }.onFailure(Timber::e)
```

이런 식으로 예외처리까지!! + 이런식으로 굳이 mutableList를 넣을 필요가 있을까?

Reply:
반영 완료!